### ISSUE TITLE
network question

### ISSUE THREAD

**Issue by YuBicheng** [ppplinday/tensorflow-vgg16-train-and-test#8]
I haven't find dropout function in your network。

**Comment by kishoreece20** [comment]
I know the basics of transfer learning method. Transfer learning is a method where a model trained on one task will transfer knowledge across many similar problems. I like to develop a methodology for lung nodule detection using ct image. cancer or no cancer as an output
